### PR TITLE
Ozark Handlebars Extension

### PR DESCRIPTION
**Pull request by rahmanusta**

### CODE REVIEW

File: ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import javax.mvc.engine.Priorities;
+import javax.mvc.engine.ViewEngine;
+import javax.mvc.engine.ViewEngineContext;
+import javax.mvc.engine.ViewEngineException;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.ConstraintViolation;
+import java.io.*;
+import java.io.BufferedReader;import java.io.IOException;import java.io.InputStream;import java.io.InputStreamReader;import java.io.PrintWriter;import java.lang.Override;import java.lang.String;import java.util.*;
```
Comment (spericas) [inline ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java]: Why multiple imports on same line?

File: ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        return view.endsWith(".hbs");
+    }
+
+    @Override
+    public void processView(ViewEngineContext context) throws ViewEngineException {
+
+        Models models = context.getModels();
+        String viewName = context.getView();
+        if (!viewName.startsWith("/"))
+            viewName = "/" + viewName;
```
Comment (spericas) [inline ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java]: Please take a look at other engines. Views should be searched for under /WEB-INF/views now.

File: ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java (hunk 1 above)
Comment (rahmanusta) [inline ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java]: I didnt understand cause :) I will multiline it

File: ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java (hunk 2 above)
Comment (rahmanusta) [inline ext/handlebars/src/main/java/com/oracle/ozark/ext/handlebars/HandlebarsViewEngine.java]: Ok, i will update it.

**Pull request merged**